kzg/src/eip_7594.rs

```diff
@@ -403,7 +403,7 @@ fn recover_cells<
 }
 
 pub fn recover_cells_and_kzg_proofs<
-    TFr: Fr,
+    TFr: Fr + Copy,
```

Comment:
I don't know if we can put this restriction on `TFr` type - need to consult with other backends, if it is possible for them to implement Copy trait.